src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1294,15 +1294,15 @@
     <value>Type '{0}' directly or indirectly inherits '{1}' without implementing any of '{2}'. Publicly-visible types should implement the generic version to broaden usability.</value>
   </data>
   <data name="MakeTypesInternalCodeFixTitle" xml:space="preserve">
-    <value>Make all public types internal</value>
+    <value>Make the public type internal</value>
   </data>
   <data name="MakeTypesInternalDescription" xml:space="preserve">
-    <value>Different than libraries, applications aren't referenced and thus all types can be made internal.</value>
+    <value>Different than libraries, applications' members aren't typically referenced publicly, therefore types could be made internal.</value>
   </data>
   <data name="MakeTypesInternalMessage" xml:space="preserve">
-    <value>Since applications' code isn't referenced, all types can be made internal</value>
+    <value>Since an application's code is typically not referenced publicly, types can be made internal</value>
```

Comment:
NIT:  still doesn't like the message, though have not good option, how it sounds? CC @gewarren 
```suggestion
    <value>When application code isn't referenced publicly, types can be made internal</value>
```

Reply:
Because an application's API isn't typically referenced from outside the assembly, types can be made internal